### ISSUE TITLE
Error running Cake.Powershell against latest Cake versions

### ISSUE THREAD

**Issue by mejas** [SharpeRAD/Cake.Powershell#35]
I'm trying to build a project using the latest version of cake. My build definition has a dependency on Cake.Powershell, and running the build definition gives the following error:
```
Error: The assembly 'Cake.Powershell, Version=0.3.5.0, Culture=neutral, PublicKeyToken=null'
is referencing an older version of Cake.Core (0.17.0).
This assembly need to reference at least Cake.Core version 0.22.0.
Another option is to downgrade Cake to an earlier version.
It's not recommended, but you can explicitly opt-out of assembly verification
by configuring the Skip Verification setting to true
(i.e. command line parameter "--settings_skipverification=true",
envrionment variable "CAKE_SETTINGS_SKIPVERIFICATION=true",
read more about configuration at https://cakebuild.net/docs/fundamentals/configuration)
```
Are there any plans in the future to support higher versions of Cake?

EDIT: it seems that there is also a [pull request](https://github.com/SharpeRAD/Cake.Powershell/pull/34) for this specific problem. I'll keep the issue open for formal tracking.

**Comment by moskam** [comment]
Quick patch (not solution) is to change version of Cake.Core in cake packages.config to 0.21.1
But I'm waiting for the fix urgently as well.

**Comment by SharpeRAD** [comment]
Pin Cake to a lower version for now, I'll look at upgrading the projects over the weekend.

**Comment by SharpeRAD** [comment]
Resolved in v0.4.0
